### PR TITLE
Fix Cypress test linting

### PR DESCRIPTION
**Pull request by joshwooding**

### CODE REVIEW

File: packages/eslint-config-modular-app/index.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ module.exports = {
   reportUnusedDisableDirectives: true,
   overrides: [
     {
-      files: ['**/__tests__/**/*.{ts,tsx,js}', '**/*.test.{ts,tsx,js}'],
+      files: ['**/__tests__/**/*.{ts,tsx,js,jsx}', '**/*.test.{ts,tsx,js,jsx}'],
```
Comment (joshwooding) [inline packages/eslint-config-modular-app/index.js:19]: Added jsx here, didn't see a reason it was omitted?

File: packages/eslint-config-modular-app/index.js [hunk 2]
```diff
@@ -16,7 +16,8 @@ module.exports = {
...
+      excludedFiles: ['*.cy.{ts,tsx,js,jsx}'],
```
Comment (joshwooding) [inline packages/eslint-config-modular-app/index.js:20]: Excludes Cypress test files.  

`*.cy.{ts,tsx,js,jsx}` is the recommend format since Cypress 10

**Comment by origami-z** [comment]
@cristiano-belloni Can you review this please?

**Comment by cristiano-belloni** [comment]
> @cristiano-belloni Can you review this please?

in my queue!

**Review by cristiano-belloni — APPROVED** [review]

**Pull request merged**